build-scripts/rspack.cjs

```diff
@@ -102,7 +102,7 @@ const createRspackConfig = ({
         chunks: !isProdBuild
           ? // improve incremental build speed, but blows up bundle size
             new RegExp(
-              `^(?!.*(${Object.keys(entry).join("|")}|work(?:er|let))$)`
+              `^(?!(${Object.keys(entry).join("|")}|.*work(?:er|let))$)`
```

Comment:
Why are we treating production and development builds differently?  Looks like the regex is trying to accomplish the same thing for the modern build, but will give a different result for the legacy build? 😕

Reply:
We had troubles with the incremental build speed, so we found this solution but it fucks up all our chunks so for prod we still use the js lambda. Yeah you are right, maybe we should also disable this for legacyBuild